src/index.js

```diff
@@ -77,18 +67,26 @@ const toDoList = (task) => {
     }
   });
 
-  upDate.addEventListener('click', updateLocalStorage());
+  taskInput.addEventListener('dblclick', () => {
+    newList.classList.toggle('hidden');
+    if (newList.classList.contains('hidden')) {
+      taskInput.removeAttribute('readonly');
+      taskInput.focus();
+    } else {
+      taskInput.removeAttribute('readonly', 'readonly');
+      toDoTasks.description = taskInput.value;
+      updateLocalStorage();
+    }
+  });
 
   updateLocalStorage();
```

Comment:
- [ ] Once more make sure that all changes to the To-Do List should be saved in local storage.

Reply:
updateLocalStorage fixed.